README.md

```diff
@@ -12,12 +12,14 @@ main = do
 
 ## Performance
 
-```
+```bash
 > cabal new-run --enable-benchmarks micro -- compareWithOtherPRNGs --small
 Up to date
-compareWithOtherPRNGs/xorshift-plus_Int (THIS PACKAGE) mean 10.63 μs  ( +- 40.63 ns  )
```

Comment:
this hardware seems better than mine. Feel free to update this with your own workstation's run of this branch.